hack/verify-staging-godeps.sh

```diff
@@ -19,4 +19,4 @@ set -o nounset
 set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
-${KUBE_ROOT}/hack/update-staging-godeps.sh -d -f "$@"
+KUBE_VERBOSE=3 KUBE_RUN_COPY_OUTPUT=N ${KUBE_ROOT}/hack/update-staging-godeps.sh -d -f "$@"
```

Comment:
Nope, this was intended. Based on the other scripts (e.g. https://github.com/kubernetes/kubernetes/blob/master/hack/verify-generated-protobuf.sh#L41) VERBOSE 3 gives the right log output for CI